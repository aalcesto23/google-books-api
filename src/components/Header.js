import React, {Component} from 'react'

class Header extends Component {
  render() {
    return (
      <div className="app-container">
        <h3>
          MY WEEKEND'S ALL BOOKED
        </h3>
        <h6 className="sub-header">
          Search for your perfect weekend book and more...
        </h6>
      </div>
    )
  }
}

export default Header;