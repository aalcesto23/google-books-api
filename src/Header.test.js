import React from 'react';
import ReactDOM from 'react-dom';
import Header from './components/Header';
import renderer from 'react-test-renderer';

it('Header renders without crashing', () => {
  const headerComp = renderer.create(
    <Header />
  )
  let header = headerComp.toJSON();
  expect(header).toMatchSnapshot();
});
