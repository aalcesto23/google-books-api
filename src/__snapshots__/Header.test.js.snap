// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<div
  className="app-container"
>
  <h3>
    MY WEEKEND'S ALL BOOKED
  </h3>
  <h6
    className="sub-header"
  >
    Search for your perfect weekend book and more...
  </h6>
</div>
`;
